cmd/gobl/serve.go

```diff
@@ -42,11 +44,18 @@ func (s *serveOpts) cmd() *cobra.Command {
 	f := cmd.Flags()
 
 	f.IntVarP(&s.httpPort, "port", "p", defaultHTTPPort, "HTTP port to listen on")
+	f.StringVarP(&s.privateKeyFile, "key", "k", defaultKeyFilename, "Default private key file for signing")
```

Comment:
When researching how the `sign` command accessed to the default key filename, I notice everything was based on the command parameter and its default value. So, I decided to follow the same approach in the serve command. The user could spin up the command specifying the signing key. If none is given, then the default one will be used.

Reply:
I think that makes a lot of sense and fits with what I'd expect.